domains/wizcann.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "wizcann"
+    },
+    "record": {
+        "CNAME": "wizcann.github.io"
```

Comment:
Can't access the website because its redirecting to the is-a.dev domain you want.

To fix this, please remove the `CNAME` file from your websites repository.

Reply:
changes made as requested! @CuteDog5695

Reply:
Guess its fine now